sixtyfps_runtime/rendering_backends/qt/qt_widgets.rs

```diff
@@ -2269,6 +2269,7 @@ pub struct NativeStyleMetrics {
     pub layout_padding: Property<f32>,
     pub text_cursor_width: Property<f32>,
     pub window_background: Property<Color>,
+    pub default_text_color: Property<Brush>,
```

Comment:
I guess this should be a Color,  not a Brush

Reply:
You're right, maybe this should be the minimal type here and we do the conversion in the style application. I'll fix that.